### DISCUSSION TITLE
Import in Android Studio

### DISCUSSION THREAD

**Discussion by iThieler** [TheTimeWalker/wallpanel-android#61] [Q&A]
I've been trying to import the source code into Android Studio for days now. Unfortunately without success. Can someone explain to me what needs to be set in Android Studio? Until now I could import any source code without problems, but wallpanel-android does not want to work. What am I doing wrong?

**Comment by L1nKinc** [comment]
I'm having the same issue. 
I tried to install Android Studio and setup everything as mentioned in the Readme. But I had no success either. 
Please can someone write a more detailed explanation how to setup the project from scratch, or has an idea what is wrong? It fails at the gradle project sync with this error:
```
Build file 'C:\Users\Markus\AndroidStudioProjects\WallPanelApp\WallPanelApp\build.gradle' line: 108

A problem occurred evaluating project ':WallPanelApp'.
> Parameter specified as non-null is null: method com.android.build.gradle.internal.dsl.BaseFlavor.buildConfigField, parameter value
```

Any help is greatly appreciated!

**Comment by CobreDev** [comment]
I was able to fix this by adding the following to my `local.properties` file:

```
code=1234
hassUrl=
broker=
brokerPass=
brokerUsername=
```
